site/docs/api/create-view-transition.md

```diff
@@ -0,0 +1,39 @@
+---
+title: createViewTransition
+parent: api
+---
+
+# createViewTransition
+
+Creates a single scoped view transition name for use with [CSS View Transitions]. This avoids potential naming collisions with other containers.
+
+> 🚧&nbsp;&nbsp;Ensure your target browsers [support view transitions].
+> Vanilla-extract supports the [container query syntax][css view transition] but does not polyfill the feature in unsupported browsers.
```

Comment:
Typo here. Also, we don't actually support the syntax just yet. Until [`csstype`](https://github.com/frenic/csstype) supports it, we'd have to implement the types ourselves.

```suggestion
> Vanilla-extract supports the [view transition syntax][css view transition] but does not polyfill the feature in unsupported browsers.
```

Reply:
I believe csstype has support for the viewTransitionName property! Is that all it would need?

Reply:
Oh, I see what you are saying, there is support for the `@view-transition` property.